tests/misc/selection_tests.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
I think it would be fine (as well as consistent) to have this file in `tests/unittests`

Reply:
I didn't because it involves downloading the `quickstart` dataset (~25MB). If you're comfortable with that happening all the time, feel free to upgrade it from `misc/` to `unittests`

Reply:
Oh, I didn't see that. Looks like it's running in GitHub Actions and not taking too much time there, but separating it into another folder (like you did) is probably helpful for people running tests locally.